blog/content/post/2020-06-15-qubes_summation.md

```diff
@@ -0,0 +1,184 @@
+    * [recording](https://youtu.be/rM0vRi6qABE?t=3)
+  3. AMA session
+    * [recording](https://youtu.be/rM0vRi6qABE?t=1904)
+
+* 04.06
+  1. Piotr Król: DRTM for Qubes OS VMs
+    * [recording](https://youtu.be/pZF-jyJWTE4)
+  2. Michał Żygowski: Anti Evil Maid for Intel coreboot-based platform
+    * [recording](https://youtu.be/YE2FbFlszI4?t=9)
+  3. AMA session
+    * [recording](https://youtu.be/YE2FbFlszI4?t=1725)
+* 10.06
+  1. Frédéric Pierret: How to build Qubes? From components to operating system
+overview
+  2. Marek Marczykowski-Górecki: Operating system testing, when it itself uses
+  virtual machines
+  3. AMA session
+
+### Qubes OS future: AMD SEV and fwupd/LVFS
+
+ The first day of 'minisummit' gathered us around Qubes OS future: AMD SEV and
+fwupd/LVFS, where prelegents were discussing projects based on Qubes OS. The
+opening presetation was held by Michał Żygowski, Firmware Engineer in 3mdeb,
+Braswell SoC, PC Engines and Protectli maintainer in coreboot who has presented
+Qubes on SuperMicro M11SDV-4C-LN4F superserver platform, that has been tested
+with Qubes OS on board. Michał has introduced and explained it’s features,
+Secure Encrypted Visualisation (SEV) and Secure Memory Encryption (SME) with
+encrypted state (SEV-ES) extension and explained why SME is better than TSME.
+Afterwards, we have been introduced with status of mentioned features on Open
+Source, debating whether it is suppoorted, whether it can be used, what impact
+Qubes OS will have on the features or is it possible to run SEV-enabled guests
+using open-source tools. Michał described briefly also OVMF and SUSE Linux
+Enterprise Server support for SEV, Automatic Exits (AE) and Non-automatic Exits
+(NAE). After prelection, our specialists took the floor debating over host
+system protection, memory encryption on the BIOS level, XEN support and many
+more issues that can be found in the video above.
+
+The second prelegent was Norbert Kamiński, Junior Embedded Systems Engineer in
+3mdeb and open-source contributor, who has introduced us status of fwupd/LVFS
+support for Qubes OS. Starting with overall informations about Firmware Update
+and Linux Vendor Firmware Service, Norbert described three layers of it’s
+architecture (system, session, internet) and explained how exactly works secure
+web service used by hardware vendors to upload firmware archives. He also
+presented fwupdmgr CLI, client tool for manual update process, describing it’s
+role between LVFS database and fwupd. Inside project status, sponsored by NLNET
+foundation, we’ve heard about Qubes OS support challenges and architecture
+solutions, architecture plan of the fwupd/LVFS support for Qubes OS based on
+sys-usb, dom0 and UpdateVM, process of downloading and updating firmware and
+what is going to be done in the next steps of development: qubes fwupdmgr script
+and .cab arhives validation. After the presentation came many questions around
```

Comment:
@KBanecka maybe wrap each .cab (\`.cab\`) to get code syntax: `.cab`

Reply:
done